tooling/template-base/components/Signin/SigninPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import Logo from '../Logo/Logo';
+
+const SigninPage = () => {
+  const [userEmail, setUserEmail] = useState('');
+  const [userPassword, setUserPassword] = useState('');
+
+  const signInHandler = e => {
+    console.log('Login Mutation Here');
+    e.preventDefault();
+    (document.getElementById('Email') as HTMLInputElement).value = '';
```

Comment:
This works, but since we've got `setUserEmail` we can keep with that vs the vanilla JS.